_usage/locales.md

```diff
@@ -295,33 +302,30 @@ Variation-Selector Sensitivity (VSS) collations distinguish between
 ideographic variation selectors in Japanese collations
 `Japanese_Bushu_Kakusu_140` and `Japanese_XJIS_140`. A variation
 sequence is made up of a base character plus an additional variation
-selector. If you don\'t the select the `_VSS` option, the variation
-selector isn\'t considered in the comparison.
+selector. If you don't the select the `_vss` option, the variation
+selector isn't considered in the comparison.
 
 VSS collations will not be provided by Babelfish in the first
 release.
 
 
-#### BIN and BIN2
+#### `bin` and `bin2`
 
-A BIN2 collation sorts characters according to code point order. The
+A `bin2` collation sorts characters according to code point order. The
 byte-by-byte binary order of UTF-8 preserves Unicode code point
 order, so this is also likely to be the best-performing collation.
-If Unicode code point order works for an application, using a BIN2
+If Unicode code point order works for an application, using a `bin2`
 collation should certainly be considered, but could result in data
 being displayed on the client in an order that is culturally
 unexpected. New mappings to lower-case characters are added to
-Unicode as time progresses, so the `LOWER` function may perform
+Unicode as time progresses, so the `lower` function may perform
 in a different way on different versions of ICU. This is a special case of
 the more general collation versioning problem rather than 
-something specific to the BIN2 collation. Babelfish will
-provide the `BBF_Latin1_General_BIN2` collation with the
+something specific to the `bin2` collation. Babelfish will
+provide the `bbf_latin1_general_bin2` collation with the
```

Comment:
we might not want to refer to this as a distribution.  maybe just omit the phrase 'with the Babelfish distribution', which adds no information anyway

Reply:
Correct, done.